client/components/Recipes/AddRecipe/AddRecipePage.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import AddRecipeForm from './AddRecipeForm.jsx';
```

Comment:
You don't have to append the extension name `.jsx` to the file(s) you are importing...